### PR TITLE
Introduce API 1.29

### PR DESCRIPTION
**Pull request by onabiull**
Add API for:
1. max frame size support to ExtBRC
2. tiling preset support for vp9e

### CODE REVIEW

File: api/include/mfxbrc.h [hunk 1]
```diff
@@ -53,8 +53,19 @@ typedef struct {
...
     mfxI32 QpY;             // Frame-level Luma QP
+#if (MFX_VERSION >= 1029)
+    mfxU32 maxFrameSize;    // Max frame size in bytes (used for rePak)
+    mfxU16 maxNumRePak;     // Max number of rePak to provide MaxFrameSize (from 0 to 8)
+    mfxU8  deltaQP[8];      // deltaQP[i] is adding to QP value while i-rePak 
+    mfxU32 initial_cpb_removal_delay;
+    mfxU32 initial_cpb_removal_offset;
+    mfxU32 reserved1[6];
+    mfxU16 NumExtParam;
+    mfxExtBuffer** ExtParam;   // extention buffer list
```
Comment (mgonchar) [inline api/include/mfxbrc.h]: typo: extension

Comment (dmitry-gurulev) [reply api/include/mfxbrc.h]: Need to align style - mixing Camel, snake & Pascal notation in one struct.

File: api/include/mfxdefs.h [hunk 2]
```diff
@@ -21,7 +21,7 @@
 #define __MFXDEFS_H__
 
 #define MFX_VERSION_MAJOR 1
-#define MFX_VERSION_MINOR 28
+#define MFX_VERSION_MINOR 29
```
Comment (eistarov) [inline api/include/mfxdefs.h]: Is it ok to advance API version with undocumented changes?

File: api/include/mfxbrc.h (hunk 1 above)
Comment (eistarov) [reply api/include/mfxbrc.h]: Also, different size under #if #else branches.
Supposed to be compensated for x64 by default 8-byte (ptr size) alignment, still a dangerous practice.
Need to combine 2 U16 and add extra reserved U32.
smth like this:
    mfxU32 MaxFrameSize;    // Max frame size in bytes (used for repack)
    mfxU8  DeltaQP[8];      // DeltaQP[i] is adding to QP value while i-repack 
    mfxU32 InitialCpbRemovalDelay;
    mfxU32 InitialCpbRemovalOffset;
    mfxU32 reserved[7];
    mfxU16 MaxNumRepack;     // Max number of repack to provide MaxFrameSize (from 0 to 8)
    mfxU16 NumExtParam;
    mfxExtBuffer** ExtParam;   // extension buffer list

File: api/include/mfxdefs.h (hunk 2 above)
Comment (onabiull) [reply api/include/mfxdefs.h]: Fixed

File: api/include/mfxbrc.h (hunk 1 above)
Comment (onabiull) [reply api/include/mfxbrc.h]: Fixed

File: api/include/mfxbrc.h [hunk 3]
```diff
@@ -53,8 +53,19 @@ typedef struct {
...
     mfxI32 QpY;             // Frame-level Luma QP
+#if (MFX_VERSION >= 1029)
+    mfxU32 InitialCpbRemovalDelay;
+    mfxU32 InitialCpbRemovalOffset;
+    mfxU32 reserved1[7];
+    mfxU32 MaxFrameSize;    // Max frame size in bytes (used for rePak)
+    mfxU8  DeltaQP[8];      // deltaQP[i] is adding to QP value while i-rePak
+    mfxU16 MaxNumRepak;     // Max number of rePak to provide MaxFrameSize (from 0 to 8)
+    mfxU16 NumExtParam;
+    mfxExtBuffer** ExtParam;   // extension buffer list
```
Comment (mgonchar) [inline api/include/mfxbrc.h]: All other comments starts with capital letter :)

**Pull request merged**